src/community/dyndimension/src/main/java/org/geoserver/web/data/resource/DynamicDimensionsTabPanel.java

```diff
@@ -272,12 +273,10 @@ public void validate(IValidatable<DefaultValueConfigurations> validatable) {
 
         @Override
         public void convertInput() {
-            visitChildren(TextArea.class, new org.apache.wicket.Component.IVisitor<TextArea<?>>() {
-
-                @Override
-                public Object component(TextArea<?> component) {
-                    component.updateModel();
-                    return null;
+            visitChildren(TextArea.class, (Component component, final IVisit<Void> visit) -> {
```

Comment:
Found out the lambda conversion could be less verbose and just use (component, visit), the compiler seems to be able to infer the argument types on its own.
Or did you do it on purpose to help the reader?